package.json

```diff
@@ -42,13 +42,13 @@
         "nock": "^13.0.4",
         "prettier": "^2.1.2",
         "ts-jest": "^26.5.6",
-        "typescript": "^4.0.3",
-        "slugify": "^1.6.5"
+        "typescript": "^4.0.3"
     },
     "dependencies": {
         "axios": "^0.21.3",
         "debug": "^4.3.1",
         "lodash.clonedeep": "^4.5.0",
-        "slugify": "^1.6.5"
+        "slugify": "^1.6.5",
```

Comment:
Why is `slugify` moved from `devDependencies` to `dependencies`? I don't see any direct use of it in the code.

Reply:
I've noticed just now that it's a duplicate, but the question still remains: Why is it in `dependencies` and not in `devDependencies`?

Reply:
It's automatically changed by `npm` when I installed `uuid` package.

I also do not see `slugify` usage anywhere in the code. 
I think it was added to generate 1Password ids, but after some time we stoped using it and just forgot to remove it.
I'm going to delete it from dependencies at all. Nice catch!

Reply:
It was automatically managed by `npm` when I installed `uuid` package.
There is used within `builders.ts`.
As we use it directly in out code and it won't work without it we need to keep the package in `dependencies` section. Everything that's inside `devDependencies` won't be included in production build.

Reply:
Awesome. Thanks for the clarification.